File: t/PACSFramework/CS/MedicalWorkstationCustomizationDemo/Examples/CreateWorkstationFeaturesEventsViewExample/IWorkstastionFeaturesEvents.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Leadtools.Medical.Workstation.Interfaces.Views;

namespace Leadtools.Demos.Workstation.Customized
{
   interface IWorkstastionFeaturesEventsView : IWorkstationView
   {
      void AddFeatureEvent ( string featureId, object publisher ) ;
      
      event EventHandler StartListeningToeEvents ;
      event EventHandler StopListeningToeEvents ;
      
      bool CanStop
      {
         get ;
         set ;
      }
      
      bool CanStart
      {
         get ;
         set ;
      }
   }
}

File: t/PACSFramework/VB/MedicalWorkstationCustomizationDemo/Examples/CreateWorkstationFeaturesEventsViewExample/IWorkstastionFeaturesEvents.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports Leadtools.Medical.Workstation.Interfaces.Views

Namespace Leadtools.Demos.Workstation.Customized
   Interface IWorkstastionFeaturesEventsView : Inherits IWorkstationView
      Sub AddFeatureEvent(ByVal featureId As String, ByVal publisher As Object)

      Event StartListeningToeEvents As EventHandler
      Event StopListeningToeEvents As EventHandler

      Property CanStop() As Boolean

      Property CanStart() As Boolean
   End Interface
End Namespace
